smartsim/_core/mli/comm/channel/dragon_fli.py

```diff
@@ -98,10 +100,12 @@ def from_descriptor(
         cls,
         descriptor: str,
     ) -> "DragonFLIChannel":
-        """A factory method that creates an instance from a descriptor string
+        """A factory method that creates an instance from a descriptor string.
 
         :param descriptor: The descriptor that uniquely identifies the resource
-        :returns: An attached DragonFLIChannel"""
+        :returns: An attached DragonFLIChannel
+        :raises: If creation of DragonFLIChanenel fails
```

Comment:
What type of error does it raise?

Reply:
(if we raise whatever is raised by the underlying call, do we need to document it? Asking because I really don't know what we should do. Should we re-cast the Exception to something we own?)

Reply:
Oops I forgot to come back to this one! I actually don't know what error this raises, but I could raise a `SmartSimError`. As far as what to do about documenting exceptions and where, that's something I'm trying to work through in my error handling ticket as well. If we explicitly raise something here then I think we document it.